Shared/Samples/Add items to portal/AddItemsToPortalView.swift

```diff
@@ -0,0 +1,269 @@
+                }
+            } else {
+                portalItemList
+            }
+        }
+        .authenticator(authenticator)
+        .errorAlert(presentingError: $error)
+        .toolbar {
+            ToolbarItem(placement: .topBarTrailing) {
+                Button("Add Portal Item", systemImage: "plus") {
+                    Task {
+                        do {
+                            try await addPortalItem()
+                        } catch {
+                            self.error = error
+                        }
+                    }
+                }
+                .disabled(portalUser == nil)
+            }
+        }
+        .task {
+            // Loads the portal user when the view appears.
+            do {
+                let portal = Portal.arcGISOnline(connection: .authenticated)
+                try await portal.load()
+                portalUser = portal.user
+                try await fetchPortalItems()
+            } catch {
+                self.error = error
+            }
+        }
+        .onAppear {
+            // Setting the challenge handlers here in `onAppear` so user is
+            // prompted to enter credentials every time trying the sample.
+            // In real world applications, set challenge handlers at the start
+            // of the application.
+            
+            // Sets authenticator as ArcGIS and Network challenge handlers to
+            // handle authentication challenges.
+            ArcGISEnvironment.authenticationManager.handleChallenges(using: authenticator)
+            
+            // In real world applications, uncomment this code to persist
+            // credentials in the keychain and remove `signOut()` from `onDisappear`.
+            // Task { try await setupPersistentCredentialStorage() }
+            
+            // Temporarily unsets the API key for this sample to use OAuth.
+            apiKey = ArcGISEnvironment.apiKey
+            ArcGISEnvironment.apiKey = nil
+        }
+        .onDisappear {
+            // Resetting the challenge handlers and clearing credentials here in
+            // `onDisappear` so user is prompted to enter credentials every time
+            // trying the sample. In real world applications, do these from
+            // sign out functionality of the application.
+            
+            // Resets challenge handlers.
+            ArcGISEnvironment.authenticationManager.handleChallenges(using: nil)
+            
+            // Sets the API key back to the original value.
+            ArcGISEnvironment.apiKey = apiKey
+            Task { await signOut() }
+        }
+    }
+    
+    /// A list of portal items in the user's portal.
+    @ViewBuilder private var portalItemList: some View {
+        List(portalItems, id: \.id) { portalItem in
+            PortalItemView(item: portalItem)
+                .swipeActions(edge: .trailing, allowsFullSwipe: false) {
+                    Button {
```

Comment:
```suggestion
                    Button(role: .destructive) {
```

Reply:
I chose not to use `.destructive` role as the swipe button itself doesn't perform the destructive action, rather the alert's delete button does the job. I can, however, change the swipe button's background to red. What do you think?

Reply:
I suspected that. Not a big issue if you leave as is, I just don't think I recall ever seeing a gray trash icon in swipe options.